src/test/java/io/managed/services/test/kafka/KafkaAccessMgmtTest.java

```diff
@@ -729,12 +738,56 @@ public void testDefaultAdminUserCanNotDeleteACLs() {
     @SneakyThrows
     @Test(priority = 10)
     public void testAdminUserCanChangeTheKafkaInstanceOwner() {
```

Comment:
I chose to extend this existing ownership change test instead of breaking out more test cases since I thought it would be harder to grok if values like the random topic names were shared through constants between different test cases.

Maybe it's too much of a departure from the test style though, we could alternatively move the assertions into their own test case like `testNewOwnersAclRulesDeleted()` with a dependency on this one.

Reply:
Approach looks good to me.